src/mount.ts

```diff
@@ -269,33 +299,24 @@ export function mount(
         acc: { [key: string]: Function },
         [name, slot]: [string, Slot]
       ): { [key: string]: Function } => {
-        // case of an SFC getting passed
-        if (typeof slot === 'object' && 'render' in slot && slot.render) {
-          acc[name] = slot.render
-          return acc
-        }
+        if (Array.isArray(slot)) {
+          const normalized = slot.reduce<Array<Function | VNode>>(
+            (acc, curr) => {
+              const toF = slotToFunction(curr)
+              if (isObject(curr) && 'render' in curr) {
```

Comment:
I have a hard time understanding the code. Maybe some comments would help. `toF` could probably be renamed to something more meaningful (`slotAsFunction`?).

It feels like this check is somewhat redundant with what is done in `slotToFunction`. I don't know exactly how, but it looks like this should be refactor/simplified.

Reply:
Naming definitely can be improved.

I also have a hard time understanding this, or why it works - the amount of ways you can create a Vue component (and as such, a lot) is a bit overwhelming. I can think of at least 5 different ways:

- class component
- function component
- SFC
- setup returning render function
- string (`i am a string`)
- template (`<div>template</div>`)

Throw in scoped slots, and you end up with this.

Reply:
DOne